### PR TITLE
feat: remember graph zoom level on restart

### LINKED ISSUE #801: Remember graph zoom level on close and restart
The current graph zoom level should be stored to the ApplicationSettings, but not displayed in the preferences dialog.

Implementation suggestion:
- in `ApplicationSettings` add getter and setter for zoom level
- in `QtGraphicsView` store and load this new setting

### PR DESCRIPTION
**Pull request by miltolstoy**
Resolves #801

### CODE REVIEW

**Comment by miltolstoy** [comment]
@egraether 
Please review.

File: src/lib_gui/qt/graphics/QtGraphicsView.cpp [hunk 1]
```diff
@@ -166,6 +165,8 @@ QtGraphicsView::QtGraphicsView(GraphFocusHandler* focusHandler, QWidget* parent)
 	m_legendButton->setObjectName(QStringLiteral("legend_button"));
 	m_legendButton->setToolTip(QStringLiteral("show legend"));
 	connect(m_legendButton, &QPushButton::clicked, this, &QtGraphicsView::legendClicked);
+
+	m_zoomFactor = ApplicationSettings::getInstance()->getGraphZoomLevel();
```
Comment (mlangkabel) [inline src/lib_gui/qt/graphics/QtGraphicsView.cpp:169]: You could keep setting `m_zoomFactor` in the initialization list of the constructor.

File: src/lib/settings/ApplicationSettings.cpp [hunk 2]
```diff
@@ -678,6 +678,16 @@ void ApplicationSettings::setLastFilepickerLocation(const FilePath& path)
 	setValue<std::wstring>("user/last_filepicker_location", path.wstr());
 }
 
+float ApplicationSettings::getGraphZoomLevel() const
+{
+	return getValue<float>("user/graph_zoom_level", 1);
```
Comment (mlangkabel) [inline src/lib/settings/ApplicationSettings.cpp:683]: You could use `1.0f` instead of `1` to avoid a cast to float here.

**Review by mlangkabel — APPROVED** [review]
Thanks for working on this feature! Your changes are looking good to me. I also tested these changes on Windows and it is working as expected. If you want, you can address the things I mentioned in my comments, but that is optional. Just add another commit. I will merge this pull request tomorrow!

**Review by mlangkabel — APPROVED** [review]

**Pull request merged**